api/v1alpha/istio-manifests/1.4/authz/allow-data-producers-policy.yaml

```diff
@@ -9,34 +9,46 @@ spec:
   rules:
   - to:
     - operation:
-        methods: ["GET,POST,PUT,DELETE"]
-        paths: ["/v1alpha/projects/$PROJECT_ID/accounts*"]
-    - operation:
-        methods: ["GET,POST,PUT,DELETE"]
-        paths: ["/v1alpha/projects/$PROJECT_ID/policies*"]
-    - operation:
-        methods: ["GET,POST,PUT,DELETE"]
-        paths: ["/v1alpha/projects/$PROJECT_ID/datasets*"]
-    - operation:
-        methods: ["POST"]
-        paths: ["/v1alpha/projects/$PROJECT_ID/admin*"]
-    - operation:
-        methods: ["POST"]
-        paths: ["/v1alpha/projects/$PROJECT_ID/auth:isDataProducer"]
-    - operation:
-        methods: ["GET"]
-        paths: ["/v1alpha/projects/$PROJECT_ID/products*"]
-    - operation:
-        methods: ["GET,POST"]
-        paths: ["/v1alpha/projects/$PROJECT_ID/procurements*"]
-    - operation:
-        methods: ["GET,POST"]
-        paths: ["/v1alpha/projects/$PROJECT_ID/spots*"]
-  - when:
-    - key: request.auth.claims[iss]
-      values:
-      - accounts.google.com
-      - https://accounts.google.com
-      - https://securetoken.google.com/$PROJECT_ID
+        methods:
+        - GET
+        paths:
+        - /v1alpha/projects/$PROJECT_ID/datasets?*
+        - /v1alpha/projects/$PROJECT_ID/procurements
+        - /v1alpha/projects/$PROJECT_ID/procurements?*
```

Comment:
I don't think regex (question mark '?') is an option for url paths [here](https://istio.io/v1.4/docs/reference/config/security/authorization-policy/) only wildcards '*'

Reply:
It's used as a literal. That call is only made with a request variable.

Reply:
ack